pallets/elections/src/lib.rs

```diff
@@ -117,60 +109,75 @@ pub mod pallet {
                 }
         }
     }
-
+    /// Desirable size of a committee.
+    ///
+    /// When new session is planned, first reserved validators are
+    /// added to the committee. Then remaining slots are filled from total validators list excluding
+    /// reserved validators
     #[pallet::storage]
-    #[pallet::getter(fn non_reserved_members)]
-    pub type NonReservedMembers<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
+    pub type CommitteeSize<T> = StorageValue<_, u32, ValueQuery>;
 
+    /// List of reserved validators in force from a new era.
+    ///
+    /// Can be changed via `change_validators` call that requires sudo.
     #[pallet::storage]
-    pub type MembersPerSession<T> = StorageValue<_, u32, ValueQuery>;
+    pub type NextEraReservedValidators<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
 
+    /// Current's era list of reserved validators.
+    ///
+    /// This is populated from `NextEraReservedValidators`
+    /// at the time of planning the first session of the era.
     #[pallet::storage]
-    #[pallet::getter(fn reserved_members)]
-    pub type ReservedMembers<T: Config> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
+    pub type CurrentEraValidators<T: Config> =
+        StorageValue<_, (Vec<T::AccountId>, Vec<T::AccountId>), ValueQuery>;
 
+    /// List of possible validators that are not reserved.
     #[pallet::storage]
-    pub type ErasMembers<T: Config> =
-        StorageValue<_, (Vec<T::AccountId>, Vec<T::AccountId>), ValueQuery>;
+    pub type NextEraNonReservedValidators<T: Config> =
+        StorageValue<_, Vec<T::AccountId>, ValueQuery>;
 
+    /// Count per validator, how many blocks did the validator produced.
     #[pallet::storage]
     pub type SessionValidatorBlockCount<T: Config> =
         StorageMap<_, Twox64Concat, T::AccountId, BlockCount, ValueQuery>;
 
+    /// Total possible reward per validator for the current era.
+    ///
+    /// Scaled to fit in the u32.
```

Comment:
what do you mean?

Reply:
why do we need `u32` instead of `Balances`, i.e. `u128`?

Reply:
therefore include this ratio (billion) here or above `ValidatorTotalRewards`

Reply:
the ratio can change, what is important is that it fits in the u32

Reply:
where is this scaling performed?

and well, if the ratio changes, we should update docs then; message like 'it is scaled to fit in the u32' tells you effectively nothing when you want to make an extrinsic

Reply:
but there is no extrinsic associated with this storage. All the logic is in the `impls.rs` file.

Reply:
okay then